### ISSUE TITLE
Accessibility: Inaccessible toggle controls on public notices page

### ISSUE THREAD

**Issue by scottaohara** [CityOfBoston/boston.gov#973]
# Issue:
The "Search by title" and "Filter by department" toggle controls, on the [Public Notices page](https://www.boston.gov/public-notices), are not keyboard accessible, meaning that sighted keyboard users, and users reliant on screen readers **can not access** the input fields that are hidden behind the "toggle buttons".  

A user navigating by the tab key can focus on the "archived public notices" link, and the next focus stop will be on the "apply" submit input.  This is due to the fact that while "Search by Title" and "Filter by department" are styled to look like buttons, both are merely just `div` elements, which are not inherently focusable elements, by default.

# Fix:
The "Search by title" and "Filter by Department" `div`s should be re-coded as `button` elements.

These elements should adhere to the following pattern:

```html
<button type="button" aria-expanded="true/false" aria-controls="unique_id_here">
  Text label here
</button>
<div id="unique_id_here" class="drawer ...">...</div>
```

Being `button` elements, the controls will inherently allow keyboard focus.  The `aria-expanded` values should correspond to the current state of the associated toggle content.  e.g. if the toggle content is set to open, then the `aria-expanded` attribute would be set to `true`.

The `aria-controls` attribute, pointing to a unique `id`, set to the associated panel that is shown/hidden, will provide JAWS screen reader users additional context to ensure they are aware of how to proceed to the newly exposed content, after activating the toggle button.

**Comment by reillyzlab** [comment]
Thanks @scottaohara for letting us know!

Flagging that this will be an issue on any of our listing pages that have a filter. More specifically:
- https://www.boston.gov/events
- https://www.boston.gov/news
- https://www.boston.gov/guides
- https://www.boston.gov/public-notices/archive

**Comment by BrutiAnteneh** [comment]
The issue passed as expected.
